src/v/utils/tests/BUILD

```diff
@@ -1,5 +1,19 @@
+load("//bazel:build.bzl", "redpanda_cc_library")
 load("//bazel:test.bzl", "redpanda_cc_bench", "redpanda_cc_btest", "redpanda_cc_btest_no_seastar", "redpanda_cc_gtest")
 
+redpanda_cc_library(
+    name = "random",
```

Comment:
Also make testonly please

Reply:
fixed this, and all the other instances in the tree.